cmd/kube-scheduler/app/server.go

```diff
@@ -237,15 +237,9 @@ func installMetricHandler(pathRecorderMux *mux.PathRecorderMux) {
 }
 
 // newMetricsHandler builds a metrics server from the config.
-func newMetricsHandler(config *componentconfig.KubeSchedulerConfiguration) http.Handler {
+func newMetricsHandler() http.Handler {
 	pathRecorderMux := mux.NewPathRecorderMux("kube-scheduler")
 	installMetricHandler(pathRecorderMux)
-	if config.EnableProfiling {
```

Comment:
why remove this?

Reply:
I think it's duplicated with healthz handler. Should not exist in both healthz and metrics handler.

Reply:
let's open a separate PR for that: it seems enabled by different params, did not get time to deep into the detail). 

The log part LGTM.

Reply:
OK. will update right now.